test/outfitters.js

```diff
@@ -0,0 +1,1316 @@
+/*
```

Comment:
probably this file name can be changes to tempOutfitters.js for consistency purposes.